parmed/formats/sdf.py

```diff
@@ -3,6 +3,7 @@
 extension described at https://www.cas.org/content/chemical-suppliers/example-sdf
 """
 from __future__ import print_function, division, absolute_import
+import linecache
```

Comment:
it's in standard lib: https://www.google.com/search?q=linecache&ie=utf-8&oe=utf-8